test/Examples.hs

```diff
@@ -1,90 +1,53 @@
 {-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -O1 #-}
```

Comment:
?

Reply:
It goes like three or four seconds faster and doesn’t impose any burden on the rest of the project. I figured it couldn’t hurt, and saved seconds add up.